libraries/ui/src/main/java/androidx/media3/ui/PlayerView.java

```diff
@@ -1138,6 +1138,16 @@ public void setFullscreenButtonClickListener(@Nullable FullscreenButtonClickList
     controller.setOnFullScreenModeChangedListener(componentListener);
   }
 
+  /**
+   * Sets whether the fullscreen button is fullScreenButton or minimalFullScreenButton.
+   *
+   * @param isFullscreen Whether the player is currently in fullscreen.
+   */
+  public void setFullscreenButtonState(boolean isFullscreen) {
+    Assertions.checkStateNotNull(controller);
+    controller.updateIsFullScreen(isFullscreen);
```

Comment:
Just a stylistic note - you seem to be mixing `Fullscreen` and `FullScreen`. I understand that a mixture of spellings are currently used throughout the repo, but let's align on not capitalizing "S", since "fullscreen" is one word